x-eithert/src/X/Control/Monad/Trans/Either.hs

```diff
@@ -204,3 +211,28 @@ joinErrorsEither =
 tryEitherT :: (Functor m, MonadCatch m, Exception e) => (e -> x) -> m a -> EitherT x m a
 tryEitherT handler = firstEitherT handler . newEitherT . try
 {-# INLINE tryEitherT #-}
+
+-- | Lift an 'Either' into 'Errors'.
+eitherErrors :: Either e a -> Errors e a
+eitherErrors e =
```

Comment:
n

I'd be happy to not export it, as your other functions don't mention `Errors` at all